inc/config.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Utility functions to retrieve and parse config files.
+ *
+ * @package altis-core
+ */
+
+namespace Altis\Core\Config;
+
+/**
+ * Retrieve the configuration for HM Platform.
+ *
+ * The configuration is defined by merging the defaults with the various files that allow to customise a particular
+ * installation.
+ *
+ * @return array Configuration data.
+ */
+function get_config() {
```

Comment:
Wondering if we need this to be behind a class, so it gets autoloaded. Ala `HM\Platform\Config::get()` or something?

Reply:
Yep, switching to classmap.